aviary/variable_info/variables.py

```diff
@@ -604,61 +601,72 @@ class Wing:
 
 
 class Dynamic:
-    """Dynamic mission data hierarchy"""
+    """All time-dependent variables used during mission analysis"""
+
+    class Atmosphere:
+        """Variables related to atmospheric/freestream conditions"""
 
-    class Mission:
-        # all time-dependent variables used during mission analysis
         ALTITUDE = 'altitude'
         ALTITUDE_RATE = 'altitude_rate'
-        ALTITUDE_RATE_MAX = 'altitude_rate_max'
-        BATTERY_STATE_OF_CHARGE = 'battery_state_of_charge'
-        CUMULATIVE_ELECTRIC_ENERGY_USED = 'cumulative_electric_energy_used'
         DENSITY = 'density'
+        DYNAMIC_PRESSURE = 'dynamic_pressure'
+        MACH = 'mach'
+        MACH_RATE = 'mach_rate'
+        SPEED_OF_SOUND = 'speed_of_sound'
+        STATIC_PRESSURE = 'static_pressure'
+        TEMPERATURE = 'temperature'
+        VELOCITY = 'velocity'
+        VELOCITY_RATE = 'velocity_rate'
+
+    class Mission:
+        """Variables related to the mission being flown"""
+
+        CUMULATIVE_ELECTRIC_ENERGY_USED = 'cumulative_electric_energy_used'
         DISTANCE = 'distance'
         DISTANCE_RATE = 'distance_rate'
+
+    class Vehicle:
+        """Variables that define the vehicle's state"""
```

Comment:
description renamed to
"Properties of the vehicle and states of the vehicle in a vehicle-fixed reference frame."